### ISSUE TITLE
jspm install fails

### ISSUE THREAD

**Issue by crazedVic** [nakupanda/bootstrap3-dialog#158]
jspm install bootstrap-dialog=github:nakupanda/bootstrap3-dialog.git

 warn Main entry point not found for github:nakupanda/bootstrap3-dialog@1.34.5.
    Adjust this property in the package.json or with an override, setting "main": false if this is the intention.

**Comment by nakupanda** [comment]
can anyone guide me how to fix this for jspm installation ?

**Comment by nicklasb** [comment]
Add a `"main": "dist/js/bootstrap-dialog"` to package.json, to begin with.
The problem is also the unconventional  placement of the distributed files, normally, they are just in a /dist-folder, making then accessible as jspm and the other es6 module loaders assume the other files are relative to the main files. 
Either way, a main-section is a good start.

**Comment by nicklasb** [comment]
The problem for me using SystemJS/jspm is that it for some reason suddenly won't load properly, not sure if it is something with the umd-wrapper.

**Comment by nicklasb** [comment]
Nope, it was a misdirected local jspm registry, disregard all above, it seems to work with SystemJS/jspm as is now. If I make the following imports:

import "bootstrap3-dialog";
import "bootstrap3-dialog/dist/css/bootstrap-dialog.min.css!"

It works.
